matcher/service/openreview_interface.py

```diff
@@ -96,11 +95,47 @@ def maximums(self):
 
         return self._maximums
 
+    def _get_custom_demand_edges(self):
+        '''Helper function to get all the custom demand edges'''
+        custom_demand_edges = []
+        custom_demand_invitation = self.config_note.content.get('custom_user_demand_invitation')
+        if custom_demand_invitation:
+            self.logger.debug('GET grouped edges invitation id={}'.format(custom_demand_invitation))
+            custom_demand_edges = self.client.get_grouped_edges(
+                invitation=custom_demand_invitation,
+                groupby='tail',
+                tail=self.config_note.content['match_group'],
+                select='head,weight')
+        return custom_demand_edges
+
+    def _get_custom_supply_edges(self):
+        '''Helper function to get all the custom supply edges'''
+        custom_supply_edges = []
+        custom_supply_invitation = self.config_note.content.get('custom_max_papers_invitation')
+        if custom_supply_invitation:
+            self.logger.debug('GET grouped edges invitation id={}'.format(custom_supply_invitation))
+            custom_supply_edges = self.client.get_grouped_edges(
+                invitation=custom_supply_invitation,
+                groupby='head',
+                head=self.config_note.content['match_group'],
+                select='tail,weight')
+        return custom_supply_edges
+
     @property
     def demands(self):
         if self._demands is None:
-            self._demands = [int(self.config_note.content['max_users']) for paper in self.papers]
-
+            self._demands = [int(self.config_note.content['user_demand']) for paper in self.papers]
```

Comment:
I'm not sure if we should keep backward compatibility with previous configuration. May be we can check by two field names 'user_demand' and 'max_users', what do you think?

Reply:
Sounds like a good idea. I have included a test and a fix for this.